partiql-ast/src/main/java/org/partiql/ast/DataType.java

```diff
@@ -70,80 +69,68 @@ public <R, C> R accept(@NotNull AstVisitor<R, C> visitor, C ctx) {
     }
 
     public static final int UNKNOWN = 0;
-    // TODO remove `NULL` and `MISSING` variants from DataType
-    // <absent types>
-    public static final int NULL = 1;
-    public static final int MISSING = 2;
```

Comment:
(self-review) removed `NULL` and `MISSING` from `DataType`. `IS [NOT] NULL|MISSING` handled by **different** node than `ExprIsType`.

Rest of changes in file are to fix the numbering.